content/agent/otel/configure-otel-metrics.md

```diff
@@ -1,229 +1,36 @@
-        initial_delay: 1s
-        scrapers:
-            cpu: {}
-            memory: {}
-            disk: {}
-            network: {}
-            filesystem: {}
-    processors:
-        batch: {}
-    exporters:
-        otlp_exporters:
-        - server:
-            host: <saas-host>
-            port: 443
-            authenticator: headers_setter
-            tls:
-            skip_verify: false
-    extensions:
-        headers_setter:
-        headers:
-            - action: insert
-            key: "authorization"
-            value: "your-data-plane-key-here"
-```
-
-Replace the placeholder values:
-- `<NGINX-One-Console-URL>`: The URL of your NGINX One Console instance.
-- `<your-data-plane-key-here>`: Your Data Plane access token.
-
----
-
-### Step 3: Run the Container
-Run the NGINX Agent container with the configuration file mounted.
-
-Use the following command:
-```bash
-docker run -d \
-  --name nginx-agent \
-  -v $(pwd)/nginx-agent.conf:/etc/nginx-agent/nginx-agent.conf \
-  nginx/agent:latest
-```
-
-Key options explained:
-- `-d`: Runs the container in detached mode.
-- `--name nginx-agent`: Assigns a name to the container for easy identification.
-- `-v $(pwd)/nginx-agent.conf:/etc/nginx-agent/nginx-agent.conf`: Mounts the configuration file into the container.
-
----
-
-### Step 4: Verify the Container is Running
-Check the running status of the container:
-```bash
-docker ps
-```
-
-You should see an entry for `nginx-agent`. The `STATUS` field should indicate that the container is running.
-
----
-
-### Step 5: Monitor Logs
-To ensure the container is functioning properly and communicating with NGINX One Console, monitor the container logs.
-
-Run the following command:
-```bash
-docker logs -f nginx-agent
-```
-
-Look for log entries indicating successful connection to the NGINX One Console and periodic metric transmission.
-
----
-
-### Troubleshooting
-
-1. **Container Fails to Start**: 
-   - Check the configuration file for errors.
-   - Ensure the NGINX One Console endpoint is reachable from the host.
-
-2. **No Metrics Sent**:
-   - Verify the access token is valid.
-   - Confirm network connectivity to the NGINX One Console.
-
-3. **Logs Show Errors**:
-   - Examine the logs for specific error messages.
-   - Address any permission or network-related issues.
-
----
-
-## Clean Up
-To stop and remove the container when it is no longer needed, run:
-```bash
-docker stop nginx-metrics-agent
-docker rm nginx-metrics-agent
-```
-
----
-
-## Conclusion
-Following these instructions, you can successfully run a container to connect to the NGINX One Console and send metrics. For further details or issues, refer to the documentation provided by NGINX or your administrator.
\ No newline at end of file
+1. Open the file: ```/var/log/nginx-agent/opentelemetry-collector-agent.log```
+2. Look for the following logs: 
+    ```vim 
+    Everything is ready. Begin running and processing data.
+    ```
```

Comment:
```suggestion
   1. Open the file: ```/var/log/nginx-agent/opentelemetry-collector-agent.log```
   2. Look for the following logs: 
      
      ```text
      Everything is ready. Begin running and processing data.
      ```
```